packages/angular-sdk-components/src/lib/_components/widget/attachment/attachment.component.ts

```diff
@@ -180,114 +143,62 @@ export class AttachmentComponent implements OnInit, OnDestroy {
       this.allowMultiple$ = this.utils.getBooleanValue(configProps.allowMultiple);
     }
 
-    this.bShowJustDelete$ = true;
     this.label$ = label;
     this.value$ = value;
     this.status = stateProps.status;
+
     this.validateMessage = this.angularPConnectData.validateMessage;
     this.extensions$ = extensions;
     this.valueRef = (this.pConn$.getStateProps() as any).value;
     this.valueRef = this.valueRef.startsWith('.') ? this.valueRef.substring(1) : this.valueRef;
```

Comment:
can you add null check for this

```suggestion
    this.valueRef = this.valueRef?.startsWith('.') ? this.valueRef.substring(1) : this.valueRef;
```